.github/workflows/unit-tests.yml

```diff
@@ -95,9 +95,6 @@ jobs:
           mkdir rust/kcl-lib/bindings
           cp -r prepared-ts-rs-bindings/* rust/kcl-lib/bindings/
 
-      - run: npm run simpleserver:bg
-        if: ${{ github.event_name != 'release' && github.event_name != 'schedule' }}
-
       - name: Install Chromium Browser
         if: ${{ github.event_name != 'release' && github.event_name != 'schedule' }}
         run: npm run playwright install chromium --with-deps
```

Comment:
Just checking, do the integration tests still need a real web browser?

Reply:
Actually, I changed my mind and marked that as a draft based on [this](https://kittycadworkspace.slack.com/archives/C07A80B83FS/p1761060330432739) conversation. Ideally, the `npm-test-unit` job doesn't need Wasm at all.

Reply:
Correct. The proper way to solve this would be dependency injection and not directly importing WASM functions from that binding.

Our application today does not do dependency injection so the dependency must exist for the unit test to import even if my unit test is not directly using it. The import/export builder step will crash.